reflex/experimental/vars/sequence.py

```diff
@@ -1238,11 +1239,10 @@ def __init__(
             i: The index.
             _var_data: Additional hooks and imports associated with the Var.
         """
+        element_type = typing.get_args(a._var_type)
         super(ArrayItemOperation, self).__init__(
             _var_name="",
-            _var_type=(
-                a._var_type.__args__[0] if hasattr(a._var_type, "__args__") else Any
-            ),
+            _var_type=element_type[0] if element_type else Any,
```

Comment:
if the `i` is actually an int, then we should use `element_type[i % len(element_type)]` so a tuple, which has index-wise args can be indexed correctly